packages/nx/src/adapter/ngcli-adapter.ts

```diff
@@ -613,6 +613,39 @@ export class NxScopedHost extends virtualFs.ScopedHost<any> {
   }
 }
 
+/**
+ * Host used by Angular CLI builders. It reads the project configurations from
+ * the project graph to access the expanded targets.
+ */
+export class NxScopedHostForBuilders extends NxScopedHost {
+  protected readMergedWorkspaceConfiguration() {
+    return zip(
+      from(createProjectGraphAsync()),
+      this.readExistingAngularJson(),
+      this.readJson<NxJsonConfiguration>('nx.json')
+    ).pipe(
+      map(([graph, angularJson, nxJson]) => {
+        const workspaceConfig = (angularJson || { projects: {} }) as any;
+        workspaceConfig.cli ??= nxJson.cli;
+        workspaceConfig.schematics ??= nxJson.generators;
+
+        for (const projectName of Object.keys(graph.nodes)) {
+          workspaceConfig.projects[projectName] ??= {
+            ...graph.nodes[projectName].data,
+          };
+        }
+
+        return workspaceConfig;
+      }),
+      catchError((err) => {
+        console.log('Unable to read angular.json');
+        console.log(err);
```

Comment:
Should these be `.log` or `.error`?

It would only really result in color formatting in the terminal I believe, but it feels like we should keep it consistent.

Reply:
This code is consistent with what currently exists. This function overrides a parent function that currently logs the error in this way. We could definitely change it though. It makes more sense what you suggest. I'll update the other function as well.